File: Alogorithms/Floyd Warshall.py
#User function template for Python

class Solution:
	def floydWarshall(self, dist):
	    
	    n = len(dist[0])
	    MAX = 10 ** 8
	    
	    for k in range(n):
	        for i in range(n):
	            for j in range(n):
	                
	                if dist[i][k] == MAX  or dist[k][j] == MAX:
	                    continue
	                
	                dist[i][j] = min( dist[i][j], dist[i][k] + dist[k][j] )

File: Alogorithms/Floyd Warshall.js

class Solution {

    floydWarshall(dist) {
        // code here
        let n = dist.length
        let MAX = 10 ** 8
        
        for(let k = 0; k < n; k++) {
            for (let i = 0; i < n; i++) {
                for (let j = 0; j < n; j++) {
                    if( dist[i][k] ==  MAX || dist[k][j] == MAX) {
                        continue
                    }
                    dist[i][j] = Math.min( dist[i][j], dist[i][k] + dist[k][j] )
                }
            } 
        }
    }
}